test/e2e/docker_helper.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	kindImage           = "byoh/node:e2e"
-	tempKubeconfigPath  = "/tmp/mgmt.conf"
+	TempKubeconfigPath  = "/tmp/mgmt.conf"
```

Comment:
Why are we changing the case ? 🤔

Reply:
I guess this is to make it a global variable to be used in `agent/host_agent_suite_test.go`.

Reply:
This issue in the previous PR was not caught as we were persisting the kubeconfig across tests. Now to remove this after each test we have to make this constant exportable.